lib/tasks/configTenantAdmin.js

```diff
@@ -11,7 +11,7 @@ const inquirer = require("inquirer");
 const { nanoid } = require("nanoid");
 const path = require("path");
 const shell = require("shelljs");
-const utils = require(path.join(__dirname, "..", "utils", "utils"));
+const utils = require("../utils/utils");
```

Comment:
in the past, we used the `path.join()` command to ensure proper platform compatibility with referencing files.

Is this no longer a concern for cross platform compatibility?

Reply:
(and by platform, I mean `windows` vs `macOS` vs `linux` etc... )

Reply:
That may have been needed in older versions of node, but it shouldn't be necessary anymore.